AsyncDisplayKit/ASTextNode.mm

```diff
@@ -410,6 +418,8 @@ + (void)drawRect:(CGRect)bounds withParameters:(ASTextNodeDrawParameters *)param
 
 - (NSObject *)drawParametersForAsyncLayer:(_ASDisplayLayer *)layer
 {
+  [self _invalidateRendererIfNeeded];
```

Comment:
@levi now I am not sure, I need some time to understand that if view/layer can be empty here.